docs/semgrep-supply-chain/getting-started.md

```diff
@@ -85,6 +85,34 @@ You can configure your CI/CD system to trigger a Semgrep Supply Chain scan whene
  </tr>
 </table>
 
+## Scan a project without lockfiles (beta)
+
+:::info
+Semgrep does not support managed scans of projects without lockfiles.
+:::
```

Comment:
Would also suggest adding a header or separate section explaining to people how to enable this feature. "To participate in this beta, reach out to sales@semgrep.com", something like that.

Reply:
@bkettle I thought only Dependency Search requires action on our end; does lockfileless require us to enable something, too?

Reply:
Lockfileless only requires using the `--enable-local-builds` flag, there is no feature flag on our side